Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -123,6 +124,7 @@ void MemoryWidget::CreateWidgets()
 
   m_find_next = new QPushButton(tr("Find &Next"));
   m_find_previous = new QPushButton(tr("Find &Previous"));
+  m_from_file = new QPushButton(tr("From &File"));
```

Comment:
This should have a more explicit name, especially if it's only used to write from file. What about something like `Value from file` or `Write file`?